concepts/language-model.md

```diff
@@ -0,0 +1,80 @@
+---
+description: Language model
+---
+
+# Language model
+
+A language model takes text input (usually a history) and outputs the next word or character.
```

Comment:
Here, people will have no idea what "a history" means.

Personally I never encountered this term in this context before reading this article.

And the target audience is people who don't know what a language model is.

Reply:
Explained. First random PDF I found on LMs use the term history: http://www.fit.vutbr.cz/research/groups/speech/servite/2010/rnnlm_mikolov.pdf